motionplan/utils_test.go

```diff
@@ -302,9 +302,10 @@ func TestPlanToPlanStepsAndGeoPoses(t *testing.T) {
 
 	for _, tc := range tcs {
 		t.Run(tc.msg, func(t *testing.T) {
-			ps, gps, err := PlanToPlanStepsAndGeoPoses(plan, baseName, tc.origin, *planRequest)
+			ps, err := PlanToPlanSteps(plan, baseName, *planRequest, nil)
```

Comment:
Good question.
We do not have any test cases were the last param of `PlanToPlanSteps` is not nil.
For these test cases, if we were to specify some non-zero pose instead of nil, our tests would fail since we end up also testing `PlanStepsToGeoPoses`.
--------------------
`PlanToPlanSteps` and `rectifyTPspacePath` do the same thing so I do not think it would add much to test this explicitly.

Reply:
In my opinion we should have at least one test that tests the `nil` last param. Otherwise, we could change the behavior of that branch & not know that we broke important functionality.

Reply:
I will update https://viam.atlassian.net/browse/RSDK-6095 since that would handle adding this test case